.github/workflows/bcr_pr_review_approver.yml

```diff
@@ -0,0 +1,19 @@
+name: Review BCR Pull Requests
+on:
+  schedule:
+    - cron: "0 * * * *"    # Run this action hourly
```

Comment:
This looks reasonably cheap to run more frequently.

Reply:
SG, updated to every 10 mins!